frontend/pages/hosts/details/cards/Software/SelfService/SelfService.tsx

```diff
@@ -465,18 +490,8 @@ const SoftwareSelfService = ({
     );
   };
 
-  const onClickFailedUpdateStatus = (s: IHostSoftware) => {
-    const lastInstall = getLastInstall(s);
-
-    if (onShowInstallDetails && lastInstall) {
-      if ("command_uuid" in lastInstall) {
-        onShowInstallDetails({ command_uuid: lastInstall.command_uuid });
-      } else if ("install_uuid" in lastInstall) {
-        onShowInstallDetails({ install_uuid: lastInstall.install_uuid });
-      } else {
-        onShowInstallDetails(undefined);
-      }
-    }
+  const onClickFailedUpdateStatus = (hostSoftware: IHostSoftware) => {
+    onShowInstallDetails(hostSoftware);
```

Comment:
@jacobshandling 
--
Flagging this, what happens if a user clicks on a failed update status for VPP app?

Reply:
I think you mentioned when we met that this refers to some kind of additional card that's a distinct area from the self service table? This handler isn't being used by any components I've updated here, so I'll restore this to its previous state and see if I can massage it into working with the other component this refers to. I've updated the argument signature for the `onShowInstallDetails` and `onShowVPPInstallDetails` methods, though, so wherever is reading these will have to be updated to make use of the new modal/methods.